.github/CODEOWNERS

```diff
@@ -2,6 +2,15 @@
 # Each line is a file pattern followed by one or more owners.
 # Order is important. The last matching pattern has the most precedence.
 
+# .github
+/.github/                         @shahmishal
+/.github/ISSUE_TEMPLATE/          @AnthonyLatsis @hborla @LucianoPAlmeida @shahmishal @xedin
+/.github/PULL_REQUEST_TEMPLATE.md @AnthonyLatsis @hborla @LucianoPAlmeida @shahmishal @xedin
+
+# docs
+/docs/HowToGuides/ @AnthonyLatsis @LucianoPAlmeida
```

Comment:
Could you please add me to this one as well?

Reply:
Sure!

Reply:
Thank you!